apps/vr-tests/src/stories/Accordion.stories.tsx

```diff
@@ -0,0 +1,149 @@
+  ))
+  .addStory('expandIconPosition="end"', () => (
+    <Accordion expandIconPosition="end" index={[0]}>
+      <AccordionItem>
+        <AccordionHeader>Opened</AccordionHeader>
+        <AccordionPanel>Visible Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader>Closed</AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+    </Accordion>
+  ))
+  .addStory('expandIcon="<Icon/>"', () => (
+    <Accordion index={[]}>
+      <AccordionItem>
+        <AccordionHeader expandIcon={<CircleRingIcon />} expandIconPosition="start">
+          Expand Icon Start
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader expandIcon={<CircleRingIcon />} expandIconPosition="end">
+          Expand Icon End
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader inline expandIcon={<CircleRingIcon />} expandIconPosition="end">
+          Expand Icon Inline End
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+    </Accordion>
+  ))
+  .addStory('icon="<Icon/>"', () => (
+    <Accordion index={[]}>
+      <AccordionItem>
+        <AccordionHeader icon={<CircleRingIcon />} expandIconPosition="start">
+          Icon Start
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader icon={<CircleRingIcon />} expandIconPosition="end">
+          Icon End
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem>
+        <AccordionHeader inline icon={<CircleRingIcon />} expandIconPosition="end">
+          Icon Inline End
+        </AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+    </Accordion>
+  ))
+  .addStory('disabled', () => (
+    <Accordion index={[]}>
+      <AccordionItem disabled>
+        <AccordionHeader>Disabled Item Opened</AccordionHeader>
+        <AccordionPanel>Visible Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem disabled>
+        <AccordionHeader>Disabled Item Closed</AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+      <AccordionItem disabled>
+        <AccordionHeader inline>Disabled Item ClosedInline</AccordionHeader>
+        <AccordionPanel>Hidden Panel</AccordionPanel>
+      </AccordionItem>
+    </Accordion>
+  ));
+
+storiesOf('Accordion', module)
+  .addDecorator(story => (
+    <Screener steps={new Screener.Steps().snapshot('normal', { cropTo: '.testWrapper' }).end()}>
+      {story()}
+    </Screener>
+  ))
+  .addDecorator(FluentProviderDecorator)
+  .addStory('Empty', () => <Accordion />);

```

Comment:
Do we really want to test this case? Is there a motivation for this?

Reply:
I was wondering that should be a bug if eventually some styling appear in the Empty state, since the Accordion by itself should not have a style. But yeah, that's controversial